packages/core-js/internals/descriptors.js

```diff
@@ -1,6 +1,6 @@
 var fails = require('../internals/fails');
 
-// Thank's IE8 for his funny defineProperty
+// Handle IE8's unusual defineProperty implementation
```

Comment:
Rather `Detect IE8's incomplete defineProperty implementation`.

Reply:
I think updated the PR to reflect this suggestion. I'm new to PRs so not totally sure if it worked